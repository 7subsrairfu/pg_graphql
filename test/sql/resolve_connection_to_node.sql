select jsonb_pretty(
    gql.resolve($$
    {
      allBlogs {
        edges {
          node {
            ownerId
            owner {
              id
            }
          }
        }
      }
    }
    $$)
);
