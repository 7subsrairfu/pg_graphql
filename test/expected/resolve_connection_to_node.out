select jsonb_pretty(
    gql.resolve($$
    {
      allBlogs {
        edges {
          node {
            ownerId
            owner {
              id
            }
          }
        }
      }
    }
    $$)
);
             jsonb_pretty             
--------------------------------------
 {                                   +
     "data": {                       +
         "allBlogs": {               +
             "edges": [              +
                 {                   +
                     "node": {       +
                         "owner": {  +
                             "id": 1 +
                         },          +
                         "ownerId": 1+
                     }               +
                 },                  +
                 {                   +
                     "node": {       +
                         "owner": {  +
                             "id": 1 +
                         },          +
                         "ownerId": 1+
                     }               +
                 },                  +
                 {                   +
                     "node": {       +
                         "owner": {  +
                             "id": 1 +
                         },          +
                         "ownerId": 1+
                     }               +
                 },                  +
                 {                   +
                     "node": {       +
                         "owner": {  +
                             "id": 2 +
                         },          +
                         "ownerId": 2+
                     }               +
                 }                   +
             ]                       +
         }                           +
     },                              +
     "errors": [                     +
     ]                               +
 }
(1 row)

